src/utils/VerifiableCredentialService.ts

```diff
@@ -43,14 +45,24 @@ export class VerifiableCredentialService {
 		const subject = sessionItem.subject;
 		const verifiedCredential: VerifiedCredential = new VerifiableCredentialBuilder(nameParts, birthDay)
 			.build();
-		const result = {
-			sub: subject,
-			nbf: now,
-			iss: this.issuer,
-			iat: now,
-			jti: randomUUID(),
-			vc: verifiedCredential,
-		};
+		let result;
+		if (process.env.USE_MOCKED) {
+			this.logger.info("VcService: USING MOCKED");
+			result = {
+				...mockVcClaims,
+				sub: subject,
+				vc: verifiedCredential,
+			};
+		} else {
+			 result = {
+				sub: subject,
+				nbf: now,
+				iss: this.issuer,
```

Comment:
How come `this.issuer` is different for mocked?

Reply:
moved it out. it is not changed for mocked VC.